tools/c7n_tencentcloud/c7n_tencentcloud/resources/mysql_backup.py

```diff
@@ -51,6 +51,9 @@ def augment(self, resources):
             items = resp["Response"]["Items"]
             field_format = self.resource_type.datetime_fields_format["Date"]
             for item in items:
+                # backups in RUNNING status don't have a proper date yet
+                if item["Status"] == "RUNNING":
```

Comment:
in terms of being more defensive about parsing this, status != 'SUCCESS' instead might be more robust for other states.

Reply:
ie, looks like FAILED would also not have this Date attribute.

Reply:
https://www.tencentcloud.com/document/api/236/15878#BackupInfo

Reply:
thanks, fixed